src/main/java/org/folio/rest/impl/DelayedTasksAPI.java

```diff
@@ -50,11 +51,11 @@ public void deleteDelayedTaskExpiredMessages(String expirationDate, String statu
   @Override
   public void postDelayedTaskRetryFailedEmails(Map<String, String> okapiHeaders,
     Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-
+    logger.debug("postDelayedTaskRetryFailedEmails::Trying to retry failed emails");
     asyncResultHandler.handle(succeededFuture(
       PostDelayedTaskRetryFailedEmailsResponse.respond202()));
 
-    logger.info("Starting email retry job");
+    logger.info("postDelayedTaskRetryFailedEmails:: Starting email retry job");
```

Comment:
I think we can remove this log now that we're logging method name and params above